content/workshops/cnd-deploy-practices.md

```diff
@@ -0,0 +1,32 @@
+---
+draft: false
+title: "Cloud Native Developer practices on Kubernetes"
```

Comment:
```suggestion
title: "Cloud Native Developer Practices on Kubernetes"
```

Reply:
thanks for the review.  changes look good to me. :-)